Failing disconnected tests because of a name mismatch

* Details: There is a mismatch between the name that snatpool is expecting and the one that generated by test.

#### OpenStack Release
Liberty - Mitaka

#### Description
This error occurs because of a name mistmatch. The log output is:

    def test_featureoff_nosegid_common_lb_net(setup_l2adjacent_test, bigip):
        icontroldriver, start_registry = handle_init_registry(bigip, FEATURE_OFF_COMMON_NET)
        service = deepcopy(NOSEGID_CREATELB)
        logcall(setup_l2adjacent_test,
                icontroldriver._common_service_handler,
                service)
        after_create_registry = register_device(bigip)
        create_uris = (set(after_create_registry.keys()) -
                       set(start_registry.keys()))
>       assert create_uris == SEG_INDEPENDENT_LB_URIS_COMMON_NET | \
            SEG_INDEPENDENT_LB_URIS | \
            NOSEG_LB_URIS
E       assert set(['https:/...?ver=11.6.1']) == set(['https://...?ver=11.6.1'])
E         Extra items in the left set:
E         'https://localhost/mgmt/tm/ltm/snatpool/~TEST_128a63ef33bc4cf891d684fad58e7f2d~TEST_128a63ef33bc4cf891d684fad58e7f2d?ver=11.6.1'
E         Extra items in the right set:
E         'https://localhost/mgmt/tm/ltm/snatpool/~Common~TEST_128a63ef33bc4cf891d684fad58e7f2d?ver=11.6.1'
E         Full diff:
E         set([u'https://localhost/mgmt/tm/ltm/snat-translation/~Common~snat-traffic-group-local-only-ce69e293-56e7-43b8-b51c-01b91d66af20_0?ver=11.6.1',
E         -      u'https://localhost/mgmt/tm/ltm/snatpool/~TEST_128a63ef33bc4cf891d684fad58e7f2d~TEST_128a63ef33bc4cf891d684fad58e7f2d?ver=11.6.1',
E         ?                                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E         +      u'https://localhost/mgmt/tm/ltm/snatpool/~Common~TEST_128a63ef33bc4cf891d684fad58e7f2d?ver=11.6.1',
E         Detailed information truncated (5 more lines), use "-vv" to show


The same log is printed out for "test_featureoff_nosegid_create_listener_common_lb_net"


#### Agent Version
8.2.0


#### Deployment
This error can be reproduced by running "f5-openstack-agent/test/functional/neutronless/disconnected_service/":
test_disconnected_service_creation.py::test_featureoff_nosegid_common_lb_net
test_disconnected_service_creation.py::test_featureoff_nosegid_create_listener_common_lb_net

--- a/test/functional/neutronless/disconnected_service/test_disconnected_service_creation.py
+++ b/test/functional/neutronless/disconnected_service/test_disconnected_service_creation.py
@@ -120,7 +120,7 @@ SEG_INDEPENDENT_LB_URIS_COMMON_NET =\
          '-ce69e293-56e7-43b8-b51c-01b91d66af20_0?ver='+tmos_version,
 
          u'https://localhost/mgmt/tm/ltm/snatpool/'
-         '~Common'
+         '~TEST_128a63ef33bc4cf891d684fad58e7f2d'
          '~TEST_128a63ef33bc4cf891d684fad58e7f2d?ver='+tmos_version,
 
          u'https://localhost/mgmt/tm/net/self/'

